.rhcicd/clowdapp.yaml

```diff
@@ -140,6 +140,41 @@ objects:
                         secretKeyRef:
                           name: policies-ui-backend-db
                           key: ${DB_SECRET_PASSWORD_KEY}
+  - apiVersion: metrics.console.redhat.com/v1alpha1
+    kind: FloorPlan
+    metadata:
+      name: policies-backend-hms
+    spec:
+      database:
+        secretName: ${FLOORIST_DB_SECRET_NAME}
```

Comment:
This is a first FloorPlan for this project, so these template parameters would need to be added.

Reply:
Updated